services/user.js

```diff
@@ -144,6 +139,11 @@ class User {
     }
 }
 
+const toISOTime = (aUser) => {
```

Comment:
This function name is a bit misleading, people may think we are converting the object into an ISO time. I suggest rename it to something like transformDateTime, transformDateTimeFormat or similar.

Reply:
updated!